standalone/main.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2020 Pilz GmbH & Co. KG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#include <string>
+#include <iostream>
+
+#include "psen_scan_v2/default_parameters.h"
+#include "psen_scan_v2/laserscan.h"
+#include "psen_scan_v2/scanner_configuration.h"
+#include "psen_scan_v2/scanner_config_builder.h"
+#include "psen_scan_v2/scanner_v2.h"
+#include "psen_scan_v2/scan_range.h"
+
+using namespace psen_scan_v2;
+using namespace psen_scan_v2::constants;
+
+/*
+ * In this section we declare all necessary configuration parameters
+ */
+const std::string HOST_IP{ "192.168.0.50" };
+const int HOST_DATA_PORT{ 55115 };
+const int HOST_CONTROL_PORT{ 55116 };
+const std::string SCANNER_IP{ "192.168.0.10" };
+const TenthOfDegree ANGLE_START{ 0 };
+const TenthOfDegree ANGLE_END{ 2750 };
```

Comment:
Couldn't this be put as degrees for easier readability? Is there a Constructor or static method `::fromDegrees()`?

```
const auto ANGLE_END = TenthOfDegree::fromDegrees(275.0);
```

Reply:
And maybe limit the example range to something smaller for more compact console output / easier readability? Please conduct a usability check with real hardware.

Reply:
I have commited a solution. I reduced the scanrange to 137°-138°, so the output will be from the center of the scan range. Reason behind this is that a user might shake their hand in front of the scanner to try and see a difference in the output rather than the side.